### PR TITLE
Announcing Scala/Spark MOOCs and Capstone

### PR DESCRIPTION
**Pull request by heathermiller**
Review by none other than the honorable @julienrf

### CODE REVIEW

**Comment by jvican** [comment]
> Review by none other than the honarable @julienrf
  
:cry:

**Comment by heathermiller** [comment]
Haha, sorry, you can review it too @jvican. I just wanted to make sure that @julienrf could add/change anything about the capstone announcement 😛

File: blog/_posts/2017-03-13-spark-mooc-capstone-live-on-coursera.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- Relational Queries and Automatic Optimization with Spark SQL's DataFrames
+- Unfiying Performance and APIs with Spark SQL's Datasets
+
+_Note this is our first time running this course material as a MOOC, so your feedback on how we can improve course is welcome! Also please be patient with us as we work out any hiccups with our new graders._
+
+[![spark-course](/resources/img/spark-course.png)](/resources/img/blog/spark-course.png)
+
+## Scala Specialization Capstone
+
+Th [Scala Specialization
```
Comment (julienrf) [inline blog/_posts/2017-03-13-spark-mooc-capstone-live-on-coursera.md]: Typo: Th**e**

File: blog/_posts/2017-03-13-spark-mooc-capstone-live-on-coursera.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+expected to exercise the concepts that they've learned through the Scala
+Specialization so far on a significant project meant to be undertaken over the
+full 6 weeks.
+
+Students are challenged to implement, almost completely from scratch, a full
+application aimed at processing and visualizing several gigabytes of climate
+data. It covers not only data analysis, but also building ways for users to
+_interactively visualize and explore_ this climate data. Students will combine
+everything that they've learned so far to process data and allow users to
+visualize it in their browsers, and to react to user input, using Scala.js.
```
Comment (julienrf) [inline blog/_posts/2017-03-13-spark-mooc-capstone-live-on-coursera.md]: I would drop the “, and to react to user input, ” part.

File: blog/_posts/2017-03-13-spark-mooc-capstone-live-on-coursera.md (hunk 1 above)
Comment (heathermiller) [inline blog/_posts/2017-03-13-spark-mooc-capstone-live-on-coursera.md]: Fixed, merci!

File: blog/_posts/2017-03-13-spark-mooc-capstone-live-on-coursera.md (hunk 2 above)
Comment (heathermiller) [inline blog/_posts/2017-03-13-spark-mooc-capstone-live-on-coursera.md]: Already discussed. Going to leave it in :)

**Pull request merged**